packages/uui-input-file/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@umbraco-ui/uui-input-file",
+  "version": "0.0.0",
+  "license": "MIT",
+  "keywords": [
+    "Umbraco",
+    "Custom elements",
+    "Web components",
+    "UI",
+    "Lit",
+    "Input File"
+  ],
+  "description": "Umbraco UI input-file component",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/umbraco/Umbraco.UI.git",
+    "directory": "packages/uui-input-file"
+  },
+  "bugs": {
+    "url": "https://github.com/umbraco/Umbraco.UI/issues"
+  },
+  "main": "./lib/index.js",
+  "module": "./lib/index.js",
+  "types": "./lib/index.d.ts",
+  "type": "module",
+  "customElements": "custom-elements.json",
+  "files": [
+    "lib/**/*.d.ts",
+    "lib/**/*.js",
+    "custom-elements.json"
+  ],
+  "dependencies": {
+    "@umbraco-ui/uui-base": "0.0.17",
+    "@umbraco-ui/uui-file-dropzone": "0.0.0"
```

Comment:
This component seems to be dependent on uui-action-bar, uui-button, uui-icon, and uui-icon-registry-essential on top of those listed here.